### PR TITLE
12 new themes for Summertime by Ketsu

### LINKED ISSUE #688: Add new themes for Summertime style by ketsu
Submission entry: http://dreamscapes.dreamwidth.org/127130.html
D, if you could confirm we have their CLA?

### PR DESCRIPTION
**Pull request by momijizukamori**
Adds the following themes:
-A Thing With Feathers
-Capricious Insect
-Dimensions
-Enough At Last
-I'm Gonna Let It Shime
-Once Is Enough
-Reason Does Not Understand
-Take It Away
-The Poet's Abstract Head
-The Very Opposite of Meaning
-Thicker Than Water
-Uncertain Inevitability
with necessary new icons and previews

Fixes #688

### CODE REVIEW

**Comment by afuna** [comment]
Hmm, I'm getting a message that the pull request can't be automatically merged. Could you check that you're on the latest code please?

**Comment by momijizukamori** [comment]
I'll try updating and rebasing! I ran updates before I did this branch and the one earlier for Planet Caravan

**Comment by afuna** [comment]
thank you!

File: styles/summertime/themes.s2 [hunk 1]
```diff
@@ -1377,6 +1378,104 @@ function Page::print_theme_stylesheet() { """
 """; }
 
 
+<<<<<<< HEAD
```
Comment (afuna) [inline styles/summertime/themes.s2]: Missed this! Thought that the merge being resolved meant that this had been resolved. Going to back out this commit. @momijizukamori please try again without these two lines :)

**Pull request merged**